func/code.func

```diff
@@ -213,3 +233,7 @@ int get_pubkey() method_id {
 int balance() method_id {
     return my_tons();
 }
+
+int get_random() method_id {
+  return rand_no;
```

Comment:
Should `unpack_data()` first, since global variables don't persist between transactions.